.github/workflows/dev-cd.yml

```diff
@@ -0,0 +1,19 @@
+name: Dev CD
+on:
+  push:
+    branches: [ 'cd/dev' ]
```

Comment:
will this eventually change to 'main'? what's the flow between this and the subsequent PR the bot opens?

Reply:
The flow is the following:
1. Flux detects a new tag,
2. Flux reflects changes of tag in code and commits it
3. Flux pushes the changes to `cd/dev`
4. GitHub actions opens a new PR from `cd/dev` to `main`
5. Human reviews/merges the PR to `main` 
6. merge to main triggers the deployment of new tag.